test/fixtures/block.json

```diff
@@ -65,7 +97,7 @@
         "merkleRoot": "8b612db17691cee1cf9c0e41416ba9b6adac91172665eea0684a11d55f19cc2b",
         "finalSaplingRoot": "3b69dbc971990f2c5cf156fb6c7f5991375715513664ed432a5f8a2e32ca7724",
         "timestamp": 1541674944,
-        "bits": "1f1f4e07",
+        "bits": "1f00ca99",
```

Comment:
Why would the difficulty have changed on this block?

Reply:
These are the values we expect in the tests and I happened to have the check commented so it always passed. Also, this was a copy paste error.